File: go/pangram/pangram.go
package pangram

import "unicode"

func IsPangram(sentence string) bool {
	charSet := make(map[rune]bool)
	for _, c := range sentence {
		c := unicode.ToLower(c)
		if c >= 'a' && c <= 'z' {
			charSet[c] = true
		}
	}

	return len(charSet) == 26
}

File: cpp/pangram/pangram.cpp
#include "pangram.h"

#include <cctype>
#include <unordered_set>
namespace pangram {
bool is_pangram(const string& word) {
    unordered_set<char> charSet;
    for (auto i = word.cbegin(); i != word.cend(); ++i) {
        char c = tolower(*i);
        if (isalpha(c)) {
            charSet.insert(c);
        }
    }
    return charSet.size() == 26;
}
}  // namespace pangram
